requirements.txt

```diff
@@ -30,7 +30,9 @@ mergedeep~=1.3
 v3io-frames~=0.10.7
 semver~=3.0
 dependency-injector~=4.41
-fsspec>=2023.1,<2023.7
+# in version 2023.9.2 fsspec canceled "auto_mkdir" in open() function
+# we required a future solution
```

Comment:
```suggestion
# upper limit due to version 2023.9.2 fsspec removing the "auto_mkdir" in the open() function. TODO to address this differently
```
also please raise a Jira on this

Reply:
https://jira.iguazeng.com/browse/ML-5235